django_afip/models.py

```diff
@@ -88,8 +88,12 @@ def check_response(response) -> None:
     This method checks if responses have an error, and raise a readable
     message.
     """
-    if response.Errors:
-        raise exceptions.AfipException(response)
+    if "Errors" in response:
+        if response.Errors:
+            raise exceptions.AfipException(response)
+    elif "errorConstancia" in response:
+        if response.errorConstancia:
```

Comment:
At this stage it seems that there is redundancy in the validations, but first the type of key for the error is determined according to the query, and then if it contains any value.